reference/supervisor.md

```diff
@@ -466,6 +466,25 @@ refer to [this section](#wb_supervisor_simulation_reset_physics).
 
 **Name**
 
+**wb\_supervisor\_node\_reset\_controller** - *resets the controller of the given robot*
+
+{[C++](cpp-api.md#cpp_node)}, {[Java](java-api.md#java_node)}, {[Python](python-api.md#python_node)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_node_reset_controller(WbNodeRef node);
+```
+
+**Description**
+
+The `wb_supervisor_node_reset_controller()` function restarts the controller of the Robot
+passed to it. If a node other than a Robot is passed to this function, it returns an error.
```

Comment:
> "it returns an error."

This function returns void, so it cannot return an error. However a warning might be displayed in the Webots console.